docs/dg/dev/internationalization-and-multi-store/adding-stores-in-a-multi-database-setup.md

```diff
@@ -0,0 +1,224 @@
+
+A minimal recipe for adding a store:
+
+**config/install/EU/setup-store.yml**
+```json
+env:
+  NEW_RELIC_ENABLED: 0
+command-timeout: 7200
+stores:
+  - { STORE-1 }
+  - { STORE-2 }
+  ...
+sections:
+  init-storage:
+    setup-search-create-sources:
+      command: "vendor/bin/console search:setup:sources -vvv --no-ansi"
+      stores: true
+  init-storages-per-store:
+    propel-migrate:
+      command: "vendor/bin/console propel:migrate -vvv --no-ansi"
+      stores: true
+  ...
+```
+
+A minimal recipe to remove a store:
+**config/install/EU/delete-store.yml**
+```
+env:
+    NEW_RELIC_ENABLED: 0
+command-timeout: 7200
+stores:
+  - { STORE-1 }
+  - { STORE-2 }
+  ...
+sections:
+    scheduler-clean:
+        scheduler-clean:
+            command: "vendor/bin/console scheduler:clean -vvv --no-ansi || true"
+            stores: true
+    clean-storage:
+        clean-storage:
+            command: "vendor/bin/console storage:delete -vvv --no-ansi"
+            stores: true
+    ...
+```
+
+You can use these custom recipes for the deployment of the application by adding them to your main deployment file. Examples:
+
+```json
+...
+SPRYKER_HOOK_DESTRUCTIVE_INSTALL: "vendor/bin/install {STORES_GO_HERE} -r EU/setup-store --no-ansi -vvv"
+...
+```
+
+```json
+...
+SPRYKER_HOOK_DESTRUCTIVE_INSTALL: "vendor/bin/install {STORES_GO_HERE} -r EU/delete-store --no-ansi -vvv"
+...
+```
+
+More information on this is provided in the following sections.
+
+### Staging setup
+
+This section describes how to roll out a new store in a staging environment.
+
+#### Staging environment configuration
+
+Add the configuration for the new store to the staging environment’s configuration.
+
+For initialize the DB, run a destructive deployment for the new store. To not affect existing stores, in the `image.environment` section of the deployment file, define only the new store in `SPRYKER_HOOK_DESTRUCTIVE_INSTALL`. In the following example, new PL and AT stores are introduced:
+```yml
+...
+SPRYKER_HOOK_DESTRUCTIVE_INSTALL: "vendor/bin/install PL,AT -r EU/destructive --no-ansi -vvv"
+...
+```
+
+You can also use a custom recipe as described in [Setting up additional deployment recipes](#setting-up-additional-deployment-recipes).
+
+#### Support requests
+* Open a support request to apply the new configuration to the environment. Attach the deploy file and shortly explain expected changes, that is the stores that need to be introduced. If the necessary configuration is in a specific repository branch, reference it in the ticket and make sure the support team has access to your code base.
+* Run the destructive deployment, assuring the right store(s) is specified.
```

Comment:
@feversocial How's this related to support?

Reply:
Running the destructive deploy - is not related to support directly. The idea is, as long as support request is satisfied, one (customer/partner) has to run a destructive deployment as a next step.